app/frontend/src/features/search/SearchResultsList.test.tsx

```diff
@@ -0,0 +1,136 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { SearchFilters } from "features/search/useSearchFilters";
+import { Map } from "maplibre-gl";
+import { UserSearchRes } from "pb/search_pb";
+import { service } from "service";
+import users from "test/fixtures/users.json";
+import hookWrapper from "test/hookWrapper";
+import { getUser } from "test/serviceMockDefaults";
+import {
+  assertErrorAlert,
+  mockConsoleError,
+  MockedService,
+  wait,
+} from "test/utils";
+
+import SearchResultsList from "./SearchResultsList";
+
+const mockSearchFiltersFactory = (filters: SearchFilters = {}) => ({
+  active: filters,
+  change: jest.fn(),
+  remove: jest.fn(),
+  apply: jest.fn(),
+});
+
+const mockHandleResultClick = jest.fn();
+jest.mock("maplibre-gl");
+const mockMapRef = { current: new Map() };
+
+const getUserMock = service.user.getUser as MockedService<
+  typeof service.user.getUser
+>;
+
+const userSearchMock = service.search.userSearch as MockedService<
+  typeof service.search.userSearch
+>;
+
+describe("SearchResultsList", () => {
+  it("Shows a user if one is selected with no search query", async () => {
+    getUserMock.mockImplementation(getUser);
+    render(
+      <SearchResultsList
+        selectedResult={1}
+        handleResultClick={mockHandleResultClick}
+        map={mockMapRef}
+        searchFilters={mockSearchFiltersFactory()}
+      />,
+      { wrapper: hookWrapper }
+    );
+
+    expect(screen.getByRole("progressbar")).toBeVisible();
+    expect(await screen.findByRole("heading", { name: users[0].name }));
+  });
+
+  it("Is blank with no selection or query", () => {
+    render(
+      <SearchResultsList
+        selectedResult={undefined}
+        handleResultClick={mockHandleResultClick}
+        map={mockMapRef}
+        searchFilters={mockSearchFiltersFactory()}
+      />,
+      { wrapper: hookWrapper }
```

Comment:
```suggestion
      { wrapper }
```
If you import the `hookWrapper` named `wrapper` instead. Same comment for the rest rather than repeating.